sgkit/window.py

```diff
@@ -0,0 +1,174 @@
+from typing import Any, Callable
+
+import dask.array as da
+import numpy as np
+from xarray import Dataset
+
+from sgkit.utils import conditional_merge_datasets
+
+# Window definition (user code)
+
+
+def window(
```

Comment:
Thinking about the API, I wonder if we should have a simple function to "remove" the windowing from a dataset? Otherwise we might expect people to pop `window_start`/`window_stop` vars?

Reply:
Yes, that would be useful. Opened #333